dags/taar_weekly.py

```diff
@@ -77,3 +175,12 @@
     ),
     dag=taar_weekly,
 )
+
+
+
+wipe_gcs_bucket >> bigquery_dump
+bigquery_dump >> extract_avro
+extract_avro >> import_bigtable
+import_bigtable >> wipe_gcs_bucket_cleanup
+wipe_gcs_bucket_cleanup >> taar_ensemble
```

Comment:
do we need a step to wipe clean the tmp bq table or partitions?

Reply:
I've just added an extra stage to drop the temp BigQuery table